indico/web/client/styles/base/_utilities.scss

```diff
@@ -77,3 +77,11 @@
 .ellipsis {
   @include ellipsis();
 }
+
+%visually-hidden {
```

Comment:
Once merged, do we want to reuse the placeholder from the new design system? Though I see that's it not exactly the same..
Also, could you please rebase this PR? Thanks!

Reply:
Yeah, once the design system is merged, I'm going to unify all these randomly added placeholders under it.

Reply:
>Also, could you please rebase this PR? Thanks!

I'll take care of it first thing on Monday.

Reply:
Done